libs/game/scenes.ts

```diff
@@ -104,6 +104,17 @@ namespace scene {
         scene.tileMap.setMap(map);
     }
 
+    /**
+     * Set whether the current Tile Map should be enabled (used) or not (ignored).
+     * @param enabled
+     */
+    export function setTileMapEnabled(enabled: boolean) {
+        const scene = game.currentScene();
+        if (!scene.tileMap)
```

Comment:
don't allocate with enabled = false?

Reply:
I suppose this shouldn't allocate in either case. (setting the map image for the first time will set enabled to true anyway)